index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="logo">
+        <h1><a href="#">My logo</a></h1>
+        <a><img src="./icons/union.jpg" alt="menu"></a>
+    </nav>
+    <section class="home-container">
+            <section class="main-info">
+              <h2>I’m Moriasi <br>glad to see <br>you!</h2>
+                <div class="my-info">
+                        <p>I'm a software developer! I can help you <br> build a product , feature or website
+                         Look <br> through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don’t <br> hesitate to contact me.
+                        </p>
```

Comment:
- [ ] OPTIONAL: I strongly do not recommend using so many `<br>`, in this size of screen it may look good, but later on desktop size, this section will be broken, Which means that the paragraph will not be responsive. I recommend to delete all `<br>` tags and to use width of the container to properly align the content inside.